src/ApiPort/ApiPort.VisualStudio/ApiPort.VisualStudio.csproj

```diff
@@ -10,7 +10,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>ApiPortVS</RootNamespace>
     <AssemblyName>ApiPort.VisualStudio</AssemblyName>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

Comment:
i think VS has a minimum of 4.6, so we don't want to rely on something later unless we have to.

Reply:
Per this document https://docs.microsoft.com/en-us/visualstudio/productinfo/vs2017-system-requirements-vs, .NET 4.6.1 is installed with VS 2017 by default. Looks like we should be good. The reason to update to 4.6.1 is to meet the minimum .NET version requirement to support .NET Standard 2.0.